openmls/src/group/mls_group/ser.rs

```diff
@@ -6,8 +6,9 @@ use serde::{
 };
 
 /// Helper struct that contains the serializable values of an `MlsGroup.
+#[deprecated(since="0.4.1", note="It is temporarily exposed, it will be private again after #245")]
```

Comment:
This is my personal preference, please let me know if you want me to just add documentation instead of deprecated.